crates/tauri-cli/src/icon.rs

```diff
@@ -40,10 +40,48 @@ struct PngEntry {
   out_path: PathBuf,
 }
 
+enum AndroidIconKind {
+  Regular,
+  Rounded,
+}
+
+struct AndroidEntries {
+  icon: Vec<(PngEntry, AndroidIconKind)>,
+  foreground: Vec<PngEntry>,
+  background: Vec<PngEntry>,
+  monochrome: Vec<PngEntry>,
+}
+
+#[derive(Deserialize)]
+struct Manifest {
```

Comment:
Just a note that we might want to generate a schema file and put it into https://schema.tauri.app/ (also some documentations would be nice)